### DISCUSSION TITLE
Multiplayer is only listed for "New and updated games"

### DISCUSSION THREAD

**Discussion by psymin** [opengaming/osgameclones#1538] [General]
In the past osgameclones.com would display the "multiplayer:" variable for each game.

Currently it only displays that variable for "New and updated games"

**Comment by cxong** [comment]
This is intentional - trying to cut down on the amount of content for each game to clean up the main page to make it more readable.
The full set of tags are shown when you click on the original game name.
The design will be undergoing further changes, happy to hear suggestions

**Comment by cxong** [comment]
See https://github.com/opengaming/osgameclones/discussions/1430 for further discussion
